### ISSUE TITLE
Question about boundary condition regarding to displacement w

### ISSUE THREAD

**Issue by jackliuzefeng** [weili101/Deep_Plates#6]
Hello,

Thank you for providing the code for your paper！It help me a lot!
in the case : Plate Deflection, i see 
func_w = lambda x: 16*((x[:,0]+1)*(x[:,0]-1)*(x[:,1]+1)*(x[:,1]-1))**2
func_u = lambda x: (x[:,0]+1)*(x[:,0]-1)*(x[:,1]+1)*(x[:,1]-1)
func_v = lambda x: (x[:,0]+1)*(x[:,0]-1)*(x[:,1]+1)*(x[:,1]-1)
Why is the w boundary condition set this way which is different from u,v. 
More specificlly, why multiply by 16 and square it,  is there a reason for the discrepancy?

Again thank you so much,
Zefengliu

**Comment by imrooki** [comment]
> 你好，
> 
> 感谢您提供论文的代码！它对我的帮助很大！ 在“板挠度”案例中，我看到 函数 `func_w = lambda x: 16*((x[:,0]+1) _(x[:,0]-1)_ (x[:,1]+1) _(x[:,1]-1))**2`、` func_u = lambda x: (x[:,0]+1)_ (x[:,0]-1) _(x[:,1]+1)_ (x[:,1]-1)` 和 ` func_v = lambda x: (x[:,0]+1) _(x[:,0]-1)_ (x[:,1]+1)*(x[:,1]-1)`。 为什么 `w` 的边界条件设置得与 `u` 和 `v` 不同？ 更具体地说，为什么要乘以 16 再平方？这种差异有什么原因吗？
> 
> 再次非常感 谢泽峰柳

Actually, I thought its nothing for this, you can choose to add this parameter or not. I am quite interested in your recent paper published in Engineering Analysis with Boundary Elements. Can you contact me?

**Comment by jackliuzefeng** [comment]
Thank you for your explanation. It's a pleasure to communicate with you. May I ask if you are the author of the paper A physics-guided neural network framework for elastic plates: Comparison of governing equations-based and energy-based approaches?

**Comment by imrooki** [comment]
> Thank you for your explanation. It's a pleasure to communicate with you. May I ask if you are the author of the paper A physics-guided neural network framework for elastic plates: Comparison of governing equations-based and energy-based approaches?

No, actually, im not the author. Just some questions about the paper. My email is 2514244110@qq.com. Thanks.

**Comment by weili101** [comment]
The square is needed for enforcing both w=0 and dw/dx=0 (dw/dy=0) at the boundaries. The 16 is just to scale the value up to be close to 1, avoiding the neural network output being too large or too small.
